<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="volWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>571</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="resetTF">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>resetTF</string>
      </property>
     </widget>
     <widget class="QSlider" name="tf_A">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>551</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="tf_D">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>551</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="tf_B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>551</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="tf_C">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>551</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="saveImage">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Save To Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reloadShader">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Reload Shader</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="openVolume">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Volume Path</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="xRes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4096</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="zRes">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4096</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="yRes">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4096</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="xSpace">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <double>4096.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="zSpace">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <double>4096.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="ySpace">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <double>4096.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dataset resolution</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Spacing between layers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadVolume">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Volume</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>volWidget</class>
   <extends>QWidget</extends>
   <header>volwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
